test_util/azure_test_driver.py

```diff
@@ -19,11 +19,22 @@
 from test_util.test_runner import integration_test
 
 
-def get_dcos_ui(master_url):
-    try:
-        return requests.get(master_url)
-    except requests.exceptions.ConnectionError:
-        pass
+def validate_env():
+    '''
+    The following environment variables must be set to ensure the template can be deployed successfully and that
+    parameters we rely on from a testing perspective are explicitly set and not relying on defaults which may change.
+    '''
+    required = ['AZURE_PARAM_linuxAdminUsername', 'AZURE_PARAM_oauthEnabled',
```

Comment:
the linuxAdminUsername doesn't seem to exist in the azuredeploy templates. Is this something from the Azure environment in general?